### ISSUE TITLE
'Collapse directories' causes multiple display of collapsed items

### ISSUE THREAD

**Issue by bovender** [atom/tree-view#956]
With the 'Collapse directories' option enabled, my collapsed directories appear many times (in fact, 5 times) in the tree view once I save any file from that project.

![Screenshot](https://picload.org/image/rdawlrrg/shutter_2016-10-09_02.png)

The above screenshot was taken after I saved the file `metadata.rb`. The `test/recipes` folder pair exists only once of course and correctly appeared only once before saving the file. The behavior is 100% reproducible on my system.

Atom 1.9.1 with tree-view 0.208.0 on Ubuntu 16.04.

**Comment by Ben3eeE** [comment]
Can you reproduce this in safe mode `atom --safe`?

**Comment by bovender** [comment]
Yes, it also occurs with `atom --safe`.

One interesting observation: It only happens with the `test/recipes` folder pair (this is a [Chef](//chef.io) cookbook). I created a dummy folder pair `foo/bar` at the same level which does not show this behavior, regardless whether there is a file `foo/bar/dummy` or not.

However, the multiplication of entries for `test/recipes` happens every time I save a file, i.e., I can easily produce 10, 15, 20 and so on entries.

So maybe this is just a peculiarity with certain folder names. Unfortunately, I don't know how to debug this further.

**Comment by Ben3eeE** [comment]
This sounds like a duplicate of https://github.com/atom/tree-view/issues/705 which was fixed in Atom 1.11.
